report_mrp_bom_matrix/README.rst

```diff
@@ -19,7 +19,7 @@ out of it.
 Usage
 =====
 
-* Go to 'Manufacturing / Products / BOM Matrix'.
+* Go to 'Manufacturing / Master Data / BOM Matrix'.
```

Comment:
You should better move the menu to 'Manufacturing / Reporting / BOM Matrix', as it is really a report.

Reply:
done